vllm/transformers_utils/config.py

```diff
@@ -521,12 +522,20 @@ def recurse_elems(elem: Any):
     config_dict["max_position_embeddings"] = config_dict.get(
         "max_position_embeddings", 128_000)
 
+    if config_dict.get("quantization") is not None:
```

Comment:
We should be specific here
```suggestion
    if config_dict.get("quantization", {}).get("qformat_weight") == "fp8_e4m3":
```